### DISCUSSION TITLE
World Cities Times

### DISCUSSION THREAD

**Discussion by lisalista123** [phaselden/FlipIt#40] [General]
Hi, sorry for the dumb question, but how do I put the international times' screensaver? I can only see the normal one right now. I downloaded the .scr file and put it into my SysWOW64 folder, so I'm not sure if there's something else I need to download or do.

**Comment by tgcentral** [comment]
C:\Windows\SysWOW64 is the correct location.

"FlipIt" should now be available in your Screen Saver Settings. If not, maybe try rebooting.

**Comment by phaselden** [comment]
Have you tried the Settings?
https://support.microsoft.com/en-us/windows/change-your-screen-saver-settings-a9dc2a0c-dc8e-9161-d270-aaccc252082a#:~:text=Go%20to%20Settings%20%3E%20Personalization%20%3E%20Lock,from%20the%20drop%2Ddown%20list.

**Reply by lisalista123** [reply]
Hi, I just went to settings and changed it to world times and it worked perfectly. Thank you!
